src/List/Extra.elm

```diff
@@ -27,7 +27,7 @@ module List.Extra exposing
 
 # Folds
 
-@docs foldl1, foldr1, indexedFoldl, indexedFoldr
+@docs Step, foldl1, foldr1, indexedFoldl, indexedFoldr, stoppableFoldl
```

Comment:
I think it's better to have the type defined as close as possible to the function that uses it. Otherwise they'll have to CTRL-F the other element when looking at the type/function.

```suggestion
@docs foldl1, foldr1, indexedFoldl, indexedFoldr, Step, stoppableFoldl
```